examples/EventListenerExample.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+require __DIR__ . '/../vendor/autoload.php';
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\HttpFactory;
+use OpenTelemetry\Contrib\Zipkin\Exporter as ZipkinExporter;
+use OpenTelemetry\EventHandler\Dispatcher;
+use OpenTelemetry\EventHandler\Events;
+use OpenTelemetry\SDK\Attributes;
+use OpenTelemetry\SDK\Trace\SpanProcessor\SimpleSpanProcessor;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+
+//Register the event listener pair, The dispatcher here is a global Class
+Dispatcher::getinstance()->listen(Events::START, 'OpenTelemetry\EventHandler\Listener\StartSpanListener::handle');
+Dispatcher::getinstance()->listen(Events::END, 'OpenTelemetry\EventHandler\Listener\EndSpanListener::handle');
+
+$zipkinExporter = new ZipkinExporter(
+    'alwaysOnZipkinExample',
+    'http://zipkin:9411/api/v2/spans',
+    new Client(),
+    new HttpFactory(),
+    new HttpFactory()
+);
+$tracerProvider =  new TracerProvider(
+    new SimpleSpanProcessor(
+        $zipkinExporter
+    )
+);
+$tracer = $tracerProvider->getTracer();
+
+echo 'Starting AlwaysOnZipkinExample';
```

Comment:
I think the example can be much simpler. A single span, using the console exporter (more like the Getting Started example)

Reply:
Okay

Reply:
this still needs to be actioned (but, the listener bits are looking good)

Reply:
still needs to be actioned... doesn't need zipkin, update message